otter/convergence/steps.py

```diff
@@ -440,14 +440,40 @@ class ChangeCLBNode(object):
 
     def as_effect(self):
         """Produce a :obj:`Effect` to modify a load balancer node."""
-        return service_request(
+        handled_codes = _clb_check_change_node_handlers.keys()
+        eff = service_request(
             ServiceType.CLOUD_LOAD_BALANCERS,
             'PUT',
             append_segments('loadbalancers', self.lb_id,
                             'nodes', self.node_id),
             data={'condition': self.condition.name,
                   'weight': self.weight},
-            success_pred=has_code(202))
+            success_pred=has_code(*handled_codes))
+        return eff.on(partial(_clb_check_change_node, self))
+
+
+def _clb_check_change_node(step, result):
+    """
+    Check to what extent a :class:`ChangeCLBNode` response was successful.
+    """
+    response, body = result
+    handler = _clb_check_change_node_handlers[response.code]
+    return handler(step, result)
+
+
+def _clb_check_change_node_retry_on_404(step, result):
+    """
+    When updating a node results in a 404, convergence should be retried.
+    """
+    return StepResult.RETRY, [{'reason': 'CLB node not found',
+                               'lb': step.lb_id,
```

Comment:
Structured info is cool :). I am assuming we should do this everywhere. Do you think `clb_id` and `clb_node_id` would be better due to rcv3 ids also?